animation_nodes/nodes/sound/MIDI_File_Parser.py

```diff
@@ -0,0 +1,55 @@
+import os
+import bpy
+from ... base_types import AnimationNode
+from ... utils.midi import MIDI_ParseFile
+
+# path : last modification, content
+cache = {}
+
+class MIDIFile(bpy.types.Node, AnimationNode):
```

Comment:
Node classes should end with `Node`. `MIDIFileParserNode`

Reply:
Enlight me on this one please

Reply:
Just rename the class to `MIDIFileParserNode`.